### PR TITLE
Component: Button

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

**Comment by kllevin** [comment]
LGTM

**Comment by smgcode** [comment]
LGTM

File: style.scss [hunk 1]
```diff
@@ -188,6 +188,8 @@
  * COMPONENTS.
  */
 
+@import "components/button";
```
Comment (smgcode) [inline style.scss]: Should this be like the other imports; <code>@import "bower_components/scally/components/button";</code>

File: components/_button.scss [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+%btn-main,
+.btn-main {
+  text-align: center;
+  background-color: $c-btn-background-colour;
+  @include to-rem(padding, $c-btn-padding-base / 2 $c-btn-padding-base);
+
+  // Apply a border
+  @if $c-btn-border {
+
+    @include to-rem(border, $c-btn-border-thickness $c-btn-border-style $c-btn-border-colour);
```
Comment (AlecRaeside) [inline components/_button.scss:129]: Could/should these 3 variables be merged into one variable?

**Comment by kllevin** [comment]
LGTM

File: components/_button.scss (hunk 2 above)
Comment (chris-pearce) [inline components/_button.scss:129]: I'd rather keep them separated.

File: style.scss (hunk 1 above)
Comment (chris-pearce) [inline style.scss]: Good catch!

**Comment by smgcode** [comment]
LGTM

File: components/_button.scss (hunk 2 above)
Comment (AlecRaeside) [inline components/_button.scss:129]: np LGTM

**Comment by chris-pearce** [comment]
This closes: #71.

**Pull request merged**